lib/usagereporter/teleport/audit.go

```diff
@@ -43,6 +43,8 @@ const (
 	// PortKubeSessionType is the session_type in tp.session.start for Kube port
 	// forwarding.
 	PortKubeSessionType = "k8s_port"
+	// SAMLIdPSessionType is the session_type tp.session.start for a SAML IdP specific web session.
+	SAMLIdPSessionType = "saml_idp_session"
```

Comment:
Note that we still create `SAMLIdPAuthAttempt` and convert those to `SessionStartEvent`, I just renamed `KindSAMLIdPSession` to `SAMLIdPSessionType` and moved it here where it is used.